docs/fixers.rst

```diff
@@ -232,7 +232,7 @@ version of ``six`` is installed.
 ``fissix`` fixers
 +++++++++++++++
 
-Some `fixers from fissix <https://fissix.readthedocs.io/en/latest/fixers.html>_`
+Some `fixers from fissix <fissix:fixers>`
```

Comment:
this link isn't working and neither are the links from the fixers

Reply:
```suggestion
Some :ref:`fixers from fissix <fissix:fixers>`
```
I think...